nearuplib/nodelib.py

```diff
@@ -207,6 +214,10 @@ def run_binary(path,
         command.extend(['--n', str(non_validators)])
     if boot_nodes:
         command.extend(['--boot-nodes', boot_nodes])
+    if fixed_shards:
+        command.extend(['--fixed-shards'])
+    if archival_nodes:
+        command.extend(['--archival-nodes'])
```

Comment:
Those should just be `command.append('--fixed-shards')` etc.

Reply:
done.